src/app/buyeroffer/updateOffer.component.html

```diff
@@ -0,0 +1,13 @@
+<section class="updateOffer">
+
+  <h3>Updating Offers</h3>
+
+  <ul class="list-group">
+    <li class="list-group-item" *ngFor="let buyeroffer of buyeroffers">
+      BuyerOffer {{buyeroffer.uri}}, value: {{buyeroffer.value}}€
```

Comment:
It is preferable to use Angular's currency pipe. `{{value | currency:'USD':true}}`